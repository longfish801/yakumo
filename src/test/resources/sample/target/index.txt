
【＃タイトル】奥深きカレーの世界
【＃サブタイトル】甘党にスパイスを
【＃公開日】2031.12.30

　ようこそ、カレーの世界へ……！
　あなたを奥深きピンクフラミンゴの森にご案内します。

