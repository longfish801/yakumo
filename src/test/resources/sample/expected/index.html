
<!doctype html>
<html lang="ja">
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.4.0/font/bootstrap-icons.css">
<style type="text/css">
<!--
// ブロック要素
body {
	font-family: "Helvetica Neue", Arial, "Hiragino Kaku Gothic ProN", "Hiragino Sans", Meiryo, sans-serif;
	letter-spacing: 0.1rem;
	line-height: 2.3rem;
	text-align: justify;
	word-wrap: break-word;
	background-color: #f0f0f0;
}
header#header {
	background-color: #f0f0f0;
}
footer#footer {
	background-color: #f0f0f0;
}
main#main {
	background-color: #ffffff;
}
nav#sidenav {
	background-color: #ffffff;
}
dl#note {
	background-color: #ccffff;
}
figure > blockquote.blockquote {
	background-color: #ffffcc;
}
figure > figcaption.blockquote-footer {
	background-color: #ffffcc;
}
pre.code {
	background-color: #f0f0f0;
}
// 目次
ul#toc > li.toc-h2 {
	padding-left: 0rem;
}
ul#toc > li.toc-h3 {
	padding-left: 1rem;
}
ul#toc > li.toc-h4 {
	padding-left: 2rem;
}
ul#toc > li.toc-h5 {
	padding-left: 3rem;
}
ul#toc > li.toc-h6 {
	padding-left: 4rem;
}
// 総目次
ul#tocAll > li.toc-h1 {
	padding-left: 0rem;
}
ul#tocAll > li.toc-h2 {
	padding-left: 1rem;
}
ul#tocAll > li.toc-h3 {
	padding-left: 2rem;
}
ul#tocAll > li.toc-h4 {
	padding-left: 3rem;
}
ul#tocAll > li.toc-h5 {
	padding-left: 4rem;
}
ul#tocAll > li.toc-h6 {
	padding-left: 5rem;
}
// インライン要素
span.attention {
	font-size: 1.7rem;
	line-height: 1.7rem;
}
span.dot {
	text-emphasis-style: dot;
	-webkit-text-emphasis-style: filled;
	-moz-text-emphasis-style: filled;
	-ms-text-emphasis-style: filled;
}
-->
</style>
<title>奥深きカレーの世界</title>
</head>
<body class="container">

<!-- ヘッダー -->
<header id="header" class="row py-3">
<div class="col-lg-8">
<h1>奥深きカレーの世界</h1>

<ul id="subtitle" class="list-unstyled fs-5">
<li>甘党にスパイスを</li>
</ul>

</div>

<div class="col-lg-4 text-end">
<ul id="extra" class="list-unstyled fs-6">
<li>公開日 2031.12.30</li>
</ul>
</div>

</header>

<div class="row justify-content-center">

<!-- 目次 -->


<main id="main" class="col-lg-8 py-3">
<!-- 本文 -->
<p>　ようこそ、カレーの世界へ&hellip;&hellip;！<br/>
　あなたを奥深きピンクフラミンゴの森にご案内します。</p>
<!-- 総目次 -->

<ul id="tocAll" class="list-unstyled">
<li class="nav-item toc-h1"><a href="curry.html" class="nav-link">おいしいカレーの作り方</a></li>
<li class="nav-item toc-h2"><a href="curry.html#id2_1" class="nav-link">はじめに</a></li>
<li class="nav-item toc-h2"><a href="curry.html#id2_2" class="nav-link">おいしいカレーはまずサラダから</a></li>
<li class="nav-item toc-h3"><a href="curry.html#id3_3" class="nav-link">カレーショップに出発進行</a></li>
<li class="nav-item toc-h4"><a href="curry.html#id4_4" class="nav-link">ベジタブルカレーは避けろ</a></li>
<li class="nav-item toc-h4"><a href="curry.html#id4_5" class="nav-link">豆のカレーは通向け</a></li>
<li class="nav-item toc-h2"><a href="curry.html#id2_6" class="nav-link">謝辞</a></li>
</ul>

<!-- 註記 -->

</main>

</div><!-- end of row -->

<!-- フッター -->
<footer id="footer" class="row justify-content-center py-3">
<nav id="navi" class="col-lg-12 text-end">
<a href="curry.html"><i class="bi bi-arrow-left-circle"></i></a>
<a href=".."><i class="bi bi-arrow-up-circle"></i></a>
<a href="curry.html"><i class="bi bi-arrow-right-circle"></i></a>
</nav>
<hr/>
</footer>

<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>

</body>
</html>
